pkg/controllers/vdb/init_db.go

```diff
@@ -65,6 +70,11 @@ type GenericDatabaseInitializer struct {
 	ConfigurationParams *vtypes.CiMap
 }
 
+type GcpSecret struct {
+	Accesskey string
```

Comment:
Depending on other comments, this may not apply anymore. But I'm including it here so I don't forget... since we use struct to unmarshal JSON, we should have JSON style comments attached with it.
```suggestion
	Accesskey string ``json:"accesskey"`
```

Reply:
This comment is obsolete now as I changed it to map[string]string{} and this struct is no more present